src/utils/api.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const gamesApi = axios.create({
+  baseURL: "https://nc-games-w79g.onrender.com/api",
```

Comment:
Really like setting up the base url here, if your api link was to change this makes updating the app much easier and keeps the code super clean! 💯